config/sites/admissions.uiowa.edu/config_split.patch.core.entity_form_display.taxonomy_term.tags.default.yml

```diff
@@ -1,8 +1,17 @@
 adding:
+  dependencies:
+    config:
+      - field.field.taxonomy_term.tags.field_image
+    module:
+      - media_library
```

Comment:
Okay, I know I said this was okay yesterday, but I'm realizing that these are in the base file, so they shouldn't be in the patch.